hack/make-bundle-vars.mk

```diff
@@ -126,9 +126,17 @@ PROMETHEUS_BUNDLE_IMG_TAG ?= main-d5fbc29
 PROMETHEUS_BUNDLE_IMG_LOCATION ?= quay.io/ocs-dev
 PROMETHEUS_BUNDLE_IMG ?= $(PROMETHEUS_BUNDLE_IMG_LOCATION)/$(PROMETHEUS_BUNDLE_IMG_NAME):$(PROMETHEUS_BUNDLE_IMG_TAG)
 
+RECIPE_BUNDLE_NAME ?= recipe
+RECIPE_BUNDLE_IMG_NAME ?= $(RECIPE_BUNDLE_NAME)-bundle
+RECIPE_BUNDLE_VERSION ?= v0.0.1
+RECIPE_BUNDLE_IMG_TAG ?= latest
+RECIPE_BUNDLE_IMG_LOCATION ?= quay.io/raghavendra_talur
```

Comment:
Shouldn't we update this to ocs-dev?

Reply:
Anyway, the CI seems to be working so, fine 😅

Reply:
currently we dont have a image pushed, We don't want to delay the PR further. I will update once we have the image.